### ISSUE TITLE
please help

### ISSUE THREAD

**Issue by vocalk** [hirosystems/explorer#1468]
Please help, I'm in trouble 
As soon as I linked the okx wallet to alex and placed the swap order, I entered the transfer order. I didn't know Stacks was this slow. So, after the swap is done, the balance becomes insufficient and the transfer order continues to be displayed as peding. About 24 hours have passed like that. I was wondering if the problem was caused by insufficient balance, so I sent a transfer from Binance to the okx wallet. And the remittance is also not in progress.

In conclusion, the problem seems to be that the remittance transaction continues to be pending. How can I solve this?

Tx ID : 0xac9d9eb4c408153f93dbd0a213916342703bb84c09fd7cb5a2ee95c946ad5322

**Comment by andresgalante** [comment]
This isn't an Explorer issue.

Please ask for community support on the #support channel in [Discord](https://discord.gg/FdhdMUrZ).
